integration_tests.py

```diff
@@ -518,6 +518,12 @@ def _test_folder_secure_func(ctx, func):
     {"name":   "util.data_object.exists.no",
      "test": lambda ctx: data_object.exists(ctx, "/tempZone/home/research-initial/testdata/doesnotexist.txt"),
      "check": lambda x: not x},
+    {"name":   "util.data_object.get_properties.default_properties",
+     "test": lambda ctx: data_object.get_properties(ctx, 12188, "irodsResc"),
+     "check": lambda x: x["DATA_NAME"] == "lorem.txt"},
+    {"name":   "util.data_object.get_properties.no_data_object",
+     "test": lambda ctx: data_object.get_properties(ctx, 1218812188, "irodsResc"),
```

Comment:
what would be a "readable" non-exist-id? for now I use doubled data_id which surely not exist

Reply:
If you want to be sure the data object ID does not exist, I think it's best to use an ID of an existing collection, resource or user. These iRODS entities use the same ID sequence as data objects, so if we have another type of iRODS entity with that ID, we can be sure it's not an ID of an existing data object.

It's probably best to add a comment about this in the integration test, since it may not be immediately obvious why this works.

Reply:
These integration tests looks like a good start :+1:  Can you also add a test / tests for the other properties. If it's hard to test a property exactly (e.g. modification time), I think just testing that something is returned should be good enough.

Reply:
Both are fixed!